deploy/crds/metal3.io_baremetalhosts_crd.yaml

```diff
@@ -0,0 +1,470 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
This file is supposed to replace https://github.com/metal3-io/baremetal-operator/pull/338/files#diff-c3596490c667247182f3661aa8f5e84f (Github did not let me comment on that file) . I don't think we need to move it, as it is obsolete.